app/(auth)/login.tsx

```diff
@@ -1,106 +1,152 @@
+import {
+  Pressable,
+  SafeAreaView,
```

Comment:
import {SafeAreaView} from "react-native-safe-area-context";
This works better

Reply:
ill work on the changes